api_spec.yaml

```diff
@@ -288,7 +288,7 @@ paths:
         - OAuth2 Recipe
       operationId: createOAuth2AccessToken
       description: |
-        Create an OAuth2 access token
+        Create an OAuth2 access token, [click here](https://app.code2flow.com/CSieLWTZG8qh) for flow diagram
```

Comment:
/appid-<appId>/recipe/oauth2/authorizationcode
 need to change to also have tenantId

Reply:
changes made